From 453058f913021e76d3256ee52402973cfdb3e194 Mon Sep 17 00:00:00 2001
From: Adrian-Ken Rueegsegger <ken@codelabs.ch>
Date: Wed, 18 Oct 2017 14:53:35 +0200
Subject: [PATCH 3/3] time: Add T_First constant

The constant can be used to initialize variables of type Time.T.

Signed-off-by: Adrian-Ken Rueegsegger <ken@codelabs.ch>
---
 common/hw-time.ads | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/common/hw-time.ads b/common/hw-time.ads
index aeda5ec..5cfa963 100644
--- a/common/hw-time.ads
+++ b/common/hw-time.ads
@@ -18,6 +18,8 @@ is
 
    type T is private;
 
+   T_First : constant T;
+
    function Now return T
    with
       Volatile_Function,
@@ -71,6 +73,8 @@ private
 
    type T is new Word64;
 
+   T_First : constant T := T'First;
+
 end HW.Time;
 
 --  vim: set ts=8 sts=3 sw=3 et:
-- 
2.20.1

